src/Elastic.Apm.NLog/readme.md

```diff
@@ -0,0 +1,34 @@
+# Elastic APM NLog Layout Renderer
+
+Allows you to add the following place holders in your NLog templates
+
+* `ElasticApmTraceId`
+* `ElasticApmTransactionId`
+
+Which will be replaced with the appropriate Elastic APM variables if available
+
+
+## How to Enable
+
+```csharp
+			var target = new MemoryTarget();
```

Comment:
nit: I think the empty spaces are not needed here - just saying because it looks a bit strange.

Reply:
It does 👍